CHANGES/8509.misc

```diff
@@ -0,0 +1 @@
+Log the exact url used during sync.
```

Comment:
We usually use past-simple in CHANGES entries, yeah?  And would it better for the casual reader to mention something like "...to make it clear what happens in the mirror-feed scenario", or some verbiage like that?